libraries/AP_Motors/examples/AP_Motors_test/run_heli_comparison.py

```diff
@@ -94,7 +95,31 @@ def get_fields(self):
 
 # ==============================================================================
 
-frame_class_lookup = {6:'Single_Heli', 11:'Dual_Heli', 13:'Heli_Quad'}
+frame_class_lookup = {6:'Single_Heli', 11:'Dual_Heli'}
```

Comment:
Because it can't do swash, so it would need some special handling for that case, this PR doesn't change anything in heli-quad so it seemed the path of least resistance.

Reply:
If you are going to change what was already there then I don't think just deleting the old parts of a test, that we are trying to standardise, is appropriate.  The special handling should be added to this PR.